src/Uno.UI.RemoteControl.VS/GlobalJsonObserver.cs

```diff
@@ -0,0 +1,209 @@
+	{
+		// Observe file changes to globalJsonPath, and only reload when "msbuild-sdks.Uno.Sdk" changes value
+		if (globalJsonPath != null)
+		{
+			var unoSdkVersion = ReadUnoSdkVersion(globalJsonPath);
+
+			_debugAction($"GlobalJsonObserver: Observing {globalJsonPath}");
+
+			_fileWatcher = new FileSystemWatcher(Path.GetDirectoryName(globalJsonPath), Path.GetFileName(globalJsonPath));
+
+			_fileWatcher.Changed += (sender, e) =>
+			{
+				_debugAction($"GlobalJsonObserver: File changed: {e.FullPath}");
+
+				try
+				{
+					var newVersion = ReadUnoSdkVersion(globalJsonPath);
+
+					if (newVersion != unoSdkVersion)
+					{
+						unoSdkVersion = newVersion;
+
+						// Reload all projects that use the Uno.SDK
+						_debugAction($"GlobalJsonObserver: Uno.Sdk version changed to {unoSdkVersion}.");
+
+						_asyncPackage.JoinableTaskFactory.Run(async () =>
+						{
+							await NotifyRestartAsync(unoSdkVersion);
+						});
+
+					}
+				}
+				catch (Exception ex)
+				{
+					_debugAction($"GlobalJsonObserver: Error reading global.json: {ex.Message}");
+				}
+			};
+
+			_fileWatcher.NotifyFilter = NotifyFilters.LastWrite |
+						NotifyFilters.Attributes |
+						NotifyFilters.Size |
+						NotifyFilters.CreationTime |
+						NotifyFilters.FileName;
+			_fileWatcher.EnableRaisingEvents = true;
+		}
+		else
+		{
+			_debugAction("GlobalJsonObserver: No global.json file found");
+		}
+	}
+
+	private async Task NotifyRestartAsync(string? unoSdkVersion)
+	{
+		await _asyncPackage.JoinableTaskFactory.SwitchToMainThreadAsync();
+
+		if (
+			await _asyncPackage.GetServiceAsync(typeof(SVsShell)) is IVsShell shell
+			&& await _asyncPackage.GetServiceAsync(typeof(SVsInfoBarUIFactory)) is IVsInfoBarUIFactory infoBarFactory)
+		{
+			var factory = new InfoBarFactory(infoBarFactory, shell);
+			var restartVSItem = new ActionBarItem { Text = "Restart Visual Studio" };
+			var moreInformationVSItem = new ActionBarItem { Text = "More information" };
+
+			var infoBar = await factory.CreateAsync(
+				new InfoBarModel(
+					$"The Uno.Sdk version changed to {unoSdkVersion} and requires restarting Visual Studio."
+					, [
+						restartVSItem,
+						moreInformationVSItem
+					]
+					, KnownMonikers.StatusError
+					, true));
+
+			if (infoBar is not null)
+			{
+				infoBar.ActionItemClicked += (s, e) =>
+				{
+					_asyncPackage.JoinableTaskFactory.Run(async () =>
+					{
+						if (e.ActionItem == restartVSItem)
+						{
+							await _asyncPackage.JoinableTaskFactory.SwitchToMainThreadAsync();
+
+							if (shell is IVsShell4 shell4)
+							{
+								((IVsShell3)shell).IsRunningElevated(out var elevated);
+								var type = elevated ? __VSRESTARTTYPE.RESTART_Elevated : __VSRESTARTTYPE.RESTART_Normal;
+								var hr = shell4.Restart((uint)type);
+							}
+						}
+						else if (e.ActionItem == moreInformationVSItem)
+						{
+							System.Diagnostics.Process.Start("https://aka.platform.uno/upgrade-uno-packages");
```

Comment:
```suggestion
							System.Diagnostics.Process.Start(new ProcessStartInfo("https://aka.platform.uno/upgrade-uno-packages") { UseShellExecute = true });
```

Reply:
`UseShellExecute` is true by default with `System.Diagnostics.Process.Start(string)`

Reply:
Thanks for the info @jeromelaban :)